pubspec.yaml

```diff
@@ -8,7 +8,8 @@ environment:
   flutter: ">=3.0.0"
 
 dependencies:
-  enough_platform_widgets: 0.6.0
+  enough_platform_widgets:
+    git: https://github.com/Enough-Software/enough_platform_widgets
```

Comment:
Why we need to apply it? Why not an official version?

Reply:
@aryan-more  Please explain help us. Thanks

Reply:
At the time of upgrading dependency there was error in enough_platform_widgets and the latest changes aren't synced/published on pub.dev.That's why i changed the source of dependency.But as of today they have published latest version with this fixes

Reply:
I have changed the source of enough_platform_widgets from direct git to pub.dev

Reply:
Ok. @aryan-more  Please rebase it

Reply:
hey @dab246 i have already done the re base